.infra/workers.ts

```diff
@@ -191,6 +191,10 @@ export const workers: Worker[] = [
     topic: 'api.v1.post-visible',
     subscription: 'api.post-changelog-added-v2',
   },
+  {
```

Comment:
We need to add the second worker as well for the deleted banner

Reply:
Well spotted accidentally removed at testing a other flow 😅